os/tools/mkbinheader.py

```diff
@@ -134,16 +146,37 @@
         print "Error : This binary priority is not valid"
         sys.exit(1)
 
+    textsize = 0
+    rosize = 0
+    datasize = 0
+    bsssize = 0
     ram_fp = open(STATIC_RAM_ESTIMATION, 'rb')
-    # First line is not used for calculating RAM size. So throw away it.
-    ram_fp.readline()
-    ram_array = ram_fp.readline()
+    line = ram_fp.readline()
+    while line:
+        words = line.split('.')
+        if len(words) > 1:
+            words = words[1].split()
+            if len(words) > 1:
+                section = words[0]
+                size = int(words[4], 16)
+                if section == 'text':
+                    textsize = size
+                if section == 'rodata':
+                    rosize = size
+                if section == 'data':
+                    datasize = size
+                if section == 'bss':
+                    bsssize = size
+                    break
+        line = ram_fp.readline()
     ram_fp.close()
     os.remove(STATIC_RAM_ESTIMATION)
-    size_array = ram_array.split('\t')
-    static_ram_size = size_array[SIZE_CMD_SUMMATION_INDEX]
-
+    textsize = roundup_power_two(textsize)
+    rosize = rosize + datasize;
```

Comment:
Please remain why rosize has rosize + datasize.

Reply:
I added comment. But I could not find a way to check for CONFIG in python file.